contents/docs/libraries/js/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: JavaScript
-sidebarTitle: JavaScript
+title: Browser JavaScript
```

Comment:
I don't love the term "Browser JavaScript" as it's clunky and even as a designer, I know the clear difference between what "JavaScript" means when compared to Node, React, etc.

But I can see some value in clarifying in the code snippets themselves, so I'd be more okay with this PR if we leave this title (and left menu links) as-is ("JavaScript") because:
- It exists in a list with all the other libraries, where it's very easy to spot other JS-based libraries like Node so it's not likely to get confused there
- We can add a callout at the top of the JS article mentioning, "If you're looking for a Node library (or whatever else)..." - and we can link to them from that page

Reply:
> I don't love the term "Browser JavaScript" as it's clunky and even as a designer, I know the clear difference between what "JavaScript" means when compared to Node, React, etc.

I agree that its clunky, but I disagree that it's common knowledge that there is a clear difference between the two. I think it's more common for backend focused eng to get confused by this, since they're don't usually work with frontend SDKs (this is based on a recent call I had with a community member)

I researched further now what other products are calling it. Maybe we can come up with a better solution?
- [Firebase](https://firebase.google.com/docs/firestore/client/libraries) splits client and server SDKs, and refers to the js client as "web" (although if you then [click into the docs](https://firebase.google.com/docs/reference/js/firestore_), they refer to it as JavaScript)
- [AWS](https://aws.amazon.com/sdk-for-javascript/) refer to it as JS, but have a clear heading of "Develop web apps" and "Develop server-side apps"
- [Twilio](https://www.twilio.com/docs/libraries), [Stripe](https://stripe.com/docs/libraries?locale=en-GB) and [Shopify](https://shopify.dev/docs/custom-storefronts/additional-sdks) do something similar as AWS

> We can add a callout at the top of the JS article mentioning, "If you're looking for a Node library (or whatever else)..." - and we can link to them from that page

I'm not a fan of this approach, since it'll be easy to miss (even if it's at the top). We should make sure users navigate the first time to the right place they need to go.

Reply:
@corywatilo @ivanagas any thoughts on renaming the title of SDK to `JavaScript web` instead of `Browser JavaScript` ? 
And then for the code snippets title we'll use `Web` instead of `Browser JavaScript`